### ISSUE TITLE
Does this library work with the `pmtiles` protocol extension for MapLibre GL?

### ISSUE THREAD

**Issue by wcedmisten** [protomaps/protomaps-themes#15]
Hello! Thanks for making this project, it seems awesome! I recently saw [this article](https://www.kschaul.com/post/2023/02/16/how-the-post-is-replacing-mapbox-with-open-source-solutions/) and I'm trying to reproduce it.

I'm trying to set up a project using protomaps and MapLibre GL.

I've generated a `.pmtiles` file through `OpenMapTiles` and  `go-pmtiles convert`, and the file seems to contain the correct content according to the inspector tool:

![image](https://user-images.githubusercontent.com/4565200/219880378-c14b0dbd-1442-47b8-ac5a-713f839c6f5d.png)

However, when I switch out the MapLibre GL style to use my `.pmtiles` file, the map doesn't render anything in the region:

![image](https://user-images.githubusercontent.com/4565200/219880587-e623f30e-93a0-4d2d-8b5c-e99240acadf7.png)

I didn't see any examples in the doc of using both the `pmtiles` extension and `protomaps-themes-base`, but are they expected to work together?

My full repo can be found [here](https://github.com/wcedmisten/map-demo), and the
code in question is [here](https://github.com/wcedmisten/map-demo/blob/main/pages/index.tsx#L17-L44)

Thanks in advance!

**Comment by bdon** [comment]
If you are generating OpenMapTiles tiles then you can use one of the styles in the OpenMapTiles GitHub organization for MapLibre, as the organization of layers and tags is different.

**Comment by wcedmisten** [comment]
Thanks so much for the help! I didn't realize the tile format used different IDs and that this library was for the protomaps format specifically.

I got this working using [a (modified) style from OpenMapTiles](https://github.com/openmaptiles/maptiler-basic-gl-style) as you suggested.

![image](https://user-images.githubusercontent.com/4565200/219974366-0eb44c3c-1e49-47c8-b5fb-02865d1ec27a.png)

If anyone else hits this issue, feel free to use this repo as a reference. I got it working using entirely self-hosted assets (including the glyphs).
https://github.com/wcedmisten/map-demo/commit/dbce923312afe3f0d3f0210dd2343c8bda1cdf5e
